src/cli/util.js

```diff
@@ -146,8 +146,8 @@ function listDifferent(context, input, options, filename) {
     if (!prettier.check(input, options)) {
       if (!context.argv["write"]) {
         context.logger.log(filename);
-        process.exitCode = 1;
       }
+      process.exitCode = 1;
```

Comment:
This one looks like an unnecessary change.

Reply:
Agree. Fixed!